app/src/main/java/id/ac/umn/pointofsales/FragmentProducts.java

```diff
@@ -42,18 +47,68 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
 
     private void createListData(){
         Product product;
-        product = new Product("Cappucino", 43000);
+        product = new Product("1","Cappucino", 43000);
         products.add(product);
 
-        product = new Product("Americano", 45000);
+        product = new Product("2","Americano", 45000);
         products.add(product);
 
 
-        product = new Product("Caramel Macchiato", 56000);
+        product = new Product("3","Caramel Macchiato", 56000);
         products.add(product);
 
-        product = new Product("Breakfast Tea", 28000);
+        product = new Product("4","Breakfast Tea", 28000);
         products.add(product);
-
     }
+
+    ProductsAdapter.FragmentCommunication communication = new ProductsAdapter.FragmentCommunication(){
+
+        @Override
+        public void onButtonClicked(String id, String name, int price, int plusMinus) {
+            Bundle bundle = new Bundle();
+
+            Product order;
+            order = new Product(id, name, price);
+
+            if(orders.size() > 0){
+                for(int i = 0; i < orders.size(); i++){
+                    if(orders.get(i).getId() == order.getId()){
+                        if(orders.get(i).getQty() > 0 && plusMinus == 1){
```

Comment:
```suggestion
                        if(orders.get(i).getQty() > 0 && plusMinus == 1){
```
umm, what is plusMinus for??